lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,17 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'drupal_id' => $data->uid,
```

Comment:
Hopefully :v: :twisted_rightwards_arrows: we can start to deprecate using Phoenix IDs sometime in the near-ish future in favor of Northstar IDs everywhere though.